### DISCUSSION TITLE
对照表排版

### DISCUSSION THREAD

**Discussion by yemo0399** [note286/xduts#83] [Q&A] [answered]
求助，怎么设置符号对照表中的位置，模板中符号和符号名称离得好近
![111](https://user-images.githubusercontent.com/128452297/226528061-a5871828-9eae-4a34-942f-a0cfce36f105.png)

word模板中是这样的
![222](https://user-images.githubusercontent.com/128452297/226528473-ba2a901b-437c-4c25-883f-be1b3641be1c.png)

**Comment by sikouhjw** [comment]
https://github.com/note286/xduts/blob/f091547688f622b5b51e4e98889305d004e8b1b4/xduts.dtx#L968-L978

**Reply by yemo0399** [reply]
谢谢！！！

**Comment by note286** [comment, marked as answer]
https://github.com/note286/xduts/discussions/40#discussioncomment-5084749

**Reply by yemo0399** [reply]
谢谢大佬！！！已解决
